hoot-core-test/src/test/cpp/hoot/core/visitors/ToEnglishTranslationVisitorTest.cpp

```diff
@@ -87,8 +87,7 @@ class ToEnglishTranslationVisitorTest : public HootTestFixture
     {
       exceptionMsg = e.what();
     }
-    CPPUNIT_ASSERT_EQUAL(
-      QString("No source languages populated.").toStdString(), exceptionMsg.toStdString());
+    HOOT_STR_EQUALS(QString("No source languages populated."), exceptionMsg);
```

Comment:
`HOOT_STR_EQUALS` allows for `QString`s that don't have to be converted to standard strings first.